plugin/acl/acl.go

```diff
@@ -71,6 +71,9 @@ RulesCheckLoop:
 			{
 				m := new(dns.Msg)
 				m.SetRcode(r, dns.RcodeRefused)
+				m = m.SetEdns0(4096, true)
```

Comment:
Can we refactor these with method chaining, like
```
m := new(xx).
             SetRcode(yy).
             SetEdns0(zz)
```

Reply:
sure! done